test_subdir/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.14)
+project(GinkgoSubdirTest LANGUAGES CXX)
+file(CREATE_LINK "${CMAKE_CURRENT_SOURCE_DIR}/.." "${CMAKE_CURRENT_BINARY_DIR}/ginkgo" SYMBOLIC)
```

Comment:
Nice way using links for that.

Reply:
Yes, but unfortunately it requires CMake >= 3.14, so we can't use that anywhere in the public part of the build system.